proto/osmosis/concentrated-liquidity/incentive_record.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+package osmosis.concentratedliquidity.v1beta1;
+
+import "cosmos_proto/cosmos.proto";
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/duration.proto";
+
+option go_package = "github.com/osmosis-labs/osmosis/v15/x/concentrated-liquidity/types";
+
+// IncentiveRecord is the high-level struct we use to deal with an independent
+// incentive being distributed on a pool. Note that PoolId, Denom, and MinUptime
+// are included in the key so we avoid storing them in state, hence the
+// distinction between IncentiveRecord and IncentiveRecordBody.
+message IncentiveRecord {
```

Comment:
Just to clarify – we're generating the object w/ proto but still only storing `IncentiveRecordBody` in state right?

Reply:
yes, there is no change in logic in the storing part, this proto struct is only used when serializing data for genesis